exasol_integration_test_docker_environment/test/test_hash_temp_dir.py

```diff
@@ -87,16 +97,16 @@ def test_directory_with_relative_paths_fixed_hash(self):
                                          use_relative_paths=True,
                                          hash_directory_names=True,
                                          hash_file_names=True)
-        hash = hasher.hash([self.test_dir1])
+        hash = hasher.hash([simple_path_mapping(self.test_dir1)])
         ascii_hash = base64.b32encode(hash).decode("ASCII")
-        self.assertEqual("VIN3VCPDX7DAC4GD37IDF4KQTCDNNH72QV5PARVGGQ4OMB4DZTLA====", ascii_hash)
+        self.assertEqual("RX5DGLU5AV6UAEZS3AE6L7WKCYOABQY7ISLX2JYX2GWJ22HH5GYQ====", ascii_hash)
```

Comment:
This changed is caused by a semantic change of the Hashing algorithm!

Before, when we added a directory, the directory itself, including itself as root-path was added to the Hasher, in this case:
`("/tmp/tmp123/test1", "/tmp/tmp123/test1")`. 
With the new implementation, the root path changes:
`("/tmp/tmp123/", "/tmp/tmp123/test1").`

Reply:
Not valid anymore, as we now hash the destination paths